src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+@import "../utils/_variables";
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: 950px;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $white-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  margin-right: 50px;
+  margin-top: 51px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__pcode {
+    font-weight: 400;
+    font-size: 10px;
+    font-style: normal;
```

Comment:
It's the default value
![image](https://user-images.githubusercontent.com/91411692/222675732-2c874b59-93fa-4eae-9add-f57c0ce9ebcb.png)

![image](https://user-images.githubusercontent.com/91411692/222675648-684def54-5cf1-4c04-b707-21d0503a2dfb.png)

```suggestion
    font-size: 10px;
```